### PR TITLE
Feature/extensions 2507 cors https oidc etc

### PR DESCRIPTION
**Pull request by jensbac**
* Enabled CORS in `VirtualService` (https://docs.solo.io/gloo-edge/latest/guides/security/cors/).
* `ExternalSecret` for `webeam-oidc` is created automatically now for all UIs.
* Completely removed configuration of outdated `SessionCache` plugin.
* Added `sniDomain` to `VirtualService`  configs (needed for TLS certs per service).
* Implemented HTTP to HTTPS redirects for all UIs and for swagger documentation.
* Switched rewrite for `/docs` to `/swagger-ui.html` to HTTP redirect because of quirk with `springfox.js`.
* Added cookie domain for auth cookies (https://docs.solo.io/gloo-edge/latest/guides/security/auth/extauth/oauth/#cookie-options).
* Added JWKS URL to configuration for the the `AuthFlowBackend` plugin.
* Updated configuration for the `AuthM2m` plugin according to the latest (changed) API of the plugin.

### CODE REVIEW

File: charts/dvpe-deployment-gloo/templates/gloo_virtualservice.yaml [hunk 1]
```diff
@@ -13,20 +14,40 @@ spec:
...
+          {{- end }}
+        allowMethods:
+          {{- range .Values.gloo.virtualservice.spec.virtualHost.cors.allowMethods }}
+          - {{ . }}
+          {{- end }}
+        allowOrigin:
+          {{- range .Values.gloo.virtualservice.spec.virtualHost.cors.allowOrigin }}
+          - {{ . | quote }}
+          {{- end }}
+        allowOriginRegex:
```
Comment (christodo) [inline charts/dvpe-deployment-gloo/templates/gloo_virtualservice.yaml:38]: Should be removed, if we not use it

File: charts/dvpe-deployment-gloo/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.1"
 description: Helm chart for installing microservices as gloo enabled VirtualService definitions.
 name: dvpe-deployment-gloo
-version: 1.2.2
+version: 1.2.3
```
Comment (christodo) [inline charts/dvpe-deployment-gloo/Chart.yaml:5]: I would use at least 1.3.0 because the changes are more than a patch

**Review by christodo — APPROVED** [review]

File: charts/dvpe-deployment-gloo/templates/gloo_virtualservice.yaml (hunk 1 above)
Comment (jensbac) [reply charts/dvpe-deployment-gloo/templates/gloo_virtualservice.yaml:38]: I'm not sure if this is required by Gloo even if empty - will test it.

File: charts/dvpe-deployment-gloo/Chart.yaml (hunk 2 above)
Comment (jensbac) [reply charts/dvpe-deployment-gloo/Chart.yaml:5]: OK, will update to 1.3.0.

**Review by herrLierb — APPROVED** [review]

**Pull request merged**